src/factories/guide.ts

```diff
@@ -0,0 +1,70 @@
+import { Container, Ticker } from 'pixi.js'
+import { rectangleFactory } from '@/factories/rectangle'
+import { FormatDate } from '@/models/guides'
+import { waitForViewport } from '@/objects'
+import { waitForApplication } from '@/objects/application'
+import { waitForConfig } from '@/objects/config'
+import { waitForCull } from '@/objects/culling'
+import { waitForFonts } from '@/objects/fonts'
+import { waitForScale } from '@/objects/scale'
+
+export type GuideFactory = Awaited<ReturnType<typeof guideFactory>>
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export async function guideFactory() {
+  const application = await waitForApplication()
+  const viewport = await waitForViewport()
+  const cull = await waitForCull()
+  const scale = await waitForScale()
+  const { styles } = await waitForConfig()
```

Comment:
Don't destructure the config. Otherwise if it changes you won't get the latest value when accessing styles